package-res/ccc/core/cartesian/axis/cart-axis.js

```diff
@@ -590,6 +590,22 @@ function cartAxis_castTitleSize(value) {
     return pvc_Size.to(value, {singleProp: pvc.BasePanel.orthogonalLength[position]});
 }
 
+function cartAxis_labelDesiredAngles(value) {
+    if(!value) {
+        return [];
+    }
+
+    if(value.constructor !== Array) {
+        value = [value];
+    }
+
+    for (var i = 0, ic = value.length; i != value.length; ++i) {
+        value[i] = def.number.to(value[i]);
```

Comment:
when value[i] is not a number, `def.number.to` is returning undefined. We should probably filter these out and push into an output array instead.